osu.Game/Overlays/DirectOverlay.cs

```diff
@@ -179,6 +179,39 @@ private void updateResultCounts()
                                     "Tag".ToQuantity(ResultAmounts.Tags);
         }
 
+        private IEnumerable<DirectPanel> createPanels(PanelDisplayStyle displayStyle, IEnumerable<BeatmapSetInfo> sets) {
```

Comment:
This would normally get picked up by CI, but since right now it's not even compiling due to missing framework changes, I'll just point this out now - we generally use BSD style braces instead of K&R, so opening brace goes on a new line here.